### DISCUSSION TITLE
Troubles with wallet

### DISCUSSION THREAD

**Discussion by xchKicker** [Chia-Network/chia-blockchain#5606] [Support]
I cant connect the wallet. I choose my wallet its loading about 20 sec and show me the window with this information: b'l\xd0\xee\xc6^w\xea\xb4\x93[k\x82\x04IG#\x83\xf9un\xad\x1d\x85\xaf\xfa\xd5j\x82\x9e\x1c\xa4\x1a'
What is the problem?

**Comment by xchKicker** [comment]
???

**Comment by xchKicker** [comment]
????

**Comment by BrandtH22** [comment]
@xchKicker , if you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
